src/schemas/genre/genre.ts

```diff
@@ -0,0 +1,3 @@
+import {z as zsu} from "zod";
```

Comment:
looks like something with a pretties here or? 🤔 
```suggestion
import { z as zsu } from "zod";
```

Reply:
I'm going to add pre-commmit hooks since this is going to be a continuous project.